docs/extensions/provider-local.md

```diff
@@ -114,9 +109,6 @@ data:
 
 This controller generates a `NetworkPolicy` which allows the control plane pods (like `kube-apiserver`) to communicate with the worker machine pods (see [`Worker` section](#worker))).
 
-In addition, it creates a `Service` named `vpn-shoot` which is only used in case the gardenlet's `ReversedVPN` feature gate is disabled.
-This `Service` enables the `vpn-seed` containers in the `kube-apiserver` pods in the seed cluster to communicate with the `vpn-shoot` pod running in the shoot cluster.
```

Comment:
This statement is removed but I don't see any provider-local adaptation.
The docs state that this `vpn-shoot` Service is only used in case `ReversedVPN` is disabled.
@rfranzke does it mean that we no longer need this https://github.com/gardener/gardener/blob/d783576ff566ab994068817962ed734d430f6187/pkg/provider-local/controller/infrastructure/actuator.go#L150-L159 ?

Reply:
It should not be needed. I deleted it.